### PR TITLE
Better handling of errors in configure script

### PR DESCRIPTION
**Pull request by alexmuller**

### CODE REVIEW

File: deploy_vcl [hunk 1]
```diff
@@ -101,8 +106,7 @@ username = ENV['FASTLY_USER']
 password = ENV['FASTLY_PASS']
 
 if username.nil? or password.nil?
-  puts "Fastly credentials are not set. Exiting."
-  exit 1
+  raise 'Fastly credentials are not in the environment set as FASTLY_USER and FASTLY_PASS'
```
Comment (rjw1) [inline deploy_vcl]: I think thats a touch clunky.
if anything we should distinguish between them and have individual errors.
"FASTLY_PASS is not set as an environment variable"
"FASTLY_USER is not set as an environment variable"

**Comment by rjw1** [comment]
These are good improvements. merge if you disagree with my comment since its not that important :)

File: deploy_vcl (hunk 1 above)
Comment (alexmuller) [inline deploy_vcl]: Updated!

**Comment by rjw1** [comment]
:+1:  merge when ready.

**Pull request merged**